### DISCUSSION TITLE
everyone mention

### DISCUSSION THREAD

**Discussion by CrazyWolf13** [ericf1/ftwbot#8] [Ideas]
Option to send a ping (@everyone) on a tweet/Story

**Comment by ericf1** [comment]
This has been implemented in the settings update. You can do s!setSettings announcement_msg_for_post-- and then follow the prompts to have the announcement say @everyone.

**Comment by CrazyWolf13** [comment]
Yes, sorry i forgot about this documentation.
But still thanks.
